repo_resource/common.py

```diff
@@ -12,17 +12,33 @@
 import sys
 import tempfile
 import warnings
+import git
+import re
+import xml.etree.ElementTree as ET
 
 from contextlib import redirect_stdout
 from pathlib import Path
 from typing import NamedTuple
 from urllib.parse import urlparse
+from multiprocessing import Pool
 
 import ssh_agent_setup
 from repo import manifest_xml
 from repo import main as repo
 
+
 CACHEDIR = Path('/tmp/repo-resource-cache')
+SHA1_PATTERN = re.compile(r'^[0-9a-f]{40}$')
+EXCLUDE_ATTRS = {'dest-branch', 'upstream'}
+DEFAULT_CHECK_JOBS = 2
```

Comment:
Duplication with check.py

Reply:
Using the variable in common everywhere now